core/object/make_virtuals.py

```diff
@@ -6,7 +6,7 @@
 template<bool required>\\
 _FORCE_INLINE_ bool _gdvirtual_##m_name##_call($CALLARGS) $CONST { \\
 	ScriptInstance *_script_instance = ((Object*)(this))->get_script_instance();\\
-	if (_script_instance) {\\
+	if (_script_instance && _script_instance->has_method(_gdvirtual_##m_name##_sn)) {\\
```

Comment:
Hm, looking at the existing code, I would have expected it to already fall through to the next check if the method didn't exist on the script. I'll need to do some testing on this.

Also, if we're now checking on the method existance up-front, we probably want to change the later code that only returns if we got `CALL_OK`: it should probably always return, regardless of the error.

Reply:
You're right, looking further, this is falling out of the second part of this patch - scene/main/node.cpp only does set_process(true) if GDVIRTUAL_IS_OVERRIDDEN returns true...but in the current state of the code, it will only ever call the GDScript process (you overrode it -> return true, no override but script -> false, no script -> look at extension).

I'm compiling locally, the two changes fixed the linked issue but I'll try with just the second.

Reply:
This change is unnecessary as you said. Locally compiled godot with just the second hunk also fixed the linked issue.